### ISSUE TITLE
Improve location of config files

### ISSUE THREAD

**Issue by diederikdehaas** [Canaan-Creative/Avalon-Management-System#10]
In you instructions you place a config file directly in `/etc/` which you shouldn't do. Create a dedicated directory for your configuration files, say something like `/etc/canaan/`.

A bit further down, you give the following direction:
`sudo cp ams-git/config/000-default.conf /etc/apache2/sites-available/000-default.conf`

Not only is `config/000-default.conf` **NOT** available in the master branch (it is in the source code download), but 'blindly' overwriting the default configuration file which is already present, is not the way to do it.
Create your own config file with a unique name and copy that to `sites-available` and then enable it with `a2ensite` would be major improvement.

**Comment by ghost** [comment]
Can you make a patch for these fixes?

**Comment by diederikdehaas** [comment]
Not anytime soon. 
I'm busy learning python, but more importantly I'm missing the connection between the `master` branch and the download. Also want proof that this repo is still worked upon and if it is indeed the software for the Avalon Controller or not.

**Comment by ghost** [comment]
Yes, it is worked on, but our communication is not good. RIght @qinfengling ?
